spec/presenters/idv/in_person/verification_results_email_presenter_spec.rb

```diff
@@ -185,40 +183,4 @@
       end
     end
   end
-
-  describe '#needs_proof_of_address?' do
-    subject(:needs_proof_of_address) { presenter.needs_proof_of_address? }
-
-    context 'with double address verification disabled' do
-      let(:capture_secondary_id_enabled) { false }
-
-      context 'with current address matching id' do
-        let(:current_address_matches_id) { true }
-
-        it { expect(needs_proof_of_address).to eq false }
-      end
-
-      context 'with current address not matching id' do
-        let(:current_address_matches_id) { false }
-
-        it { expect(needs_proof_of_address).to eq true }
-      end
-    end
-
-    context 'with double address verification enabled' do
```

Comment:
Do we need the tests below 208- just without the context?

Reply:
we don't need these anymore since we're removing `needs_proof_of_address?` from `app/presenters/idv/in_person/verification_results_email_presenter.rb`